src/lib/dfp/fill-advert-slots.ts

```diff
@@ -84,6 +84,10 @@ const fillAdvertSlots = async (): Promise<void> => {
 		.filter(
 			(adSlot) => !(isDCRMobile && adSlot.id === 'dfp-ad--top-above-nav'),
 		)
+		// We cannot guarantee that liveblog-right slots will be on the page when this code runs.
+		// liveblog-right slots are inserted by liveblog-right-column-adverts.ts, where they will be
+		// inserted when a custom event is received.
+		.filter((adSlot) => !adSlot.id.includes('liveblog-right'))
```

Comment:
Happy to do this for this AB test, but I'd be keen to avoid any extra filtering by slot here in the future.

We discussed this briefly already, but it'd be really nice if `fill-advert-slots.ts` was as greedy as possible with the slots on the page, before mounting an event listener, allowing any future slots that are inserted on the page to be picked up. I'm not sure how that'd play with Spacefinder-inserted sol